src/files-regular/index.js

```diff
@@ -1,18 +1,47 @@
 'use strict'
 
+const nodeify = require('promise-nodeify')
 const moduleConfig = require('../utils/module-config')
+const { collectify, pullify, streamify } = require('../lib/converters')
 
 module.exports = (arg) => {
   const send = moduleConfig(arg)
+  const add = require('../add')(arg)
+  const addFromFs = require('../add-from-fs')(arg)
+  const addFromURL = require('../add-from-url')(arg)
 
   return {
-    add: require('../files-regular/add')(send),
-    addReadableStream: require('../files-regular/add-readable-stream')(send),
-    addPullStream: require('../files-regular/add-pull-stream')(send),
-    addFromFs: require('../files-regular/add-from-fs')(send),
-    addFromURL: require('../files-regular/add-from-url')(send),
-    addFromStream: require('../files-regular/add')(send),
-    _addAsyncIterator: require('../files-regular/add-async-iterator')(send),
+    add: (input, options, callback) => {
+      if (typeof options === 'function') {
```

Comment:
Do these functions need to guard their inputs like this?  I think the methods they chain to also guard.  Maybe they could just be `callbackify`'d at this level?